test/units/testsuite-64.sh

```diff
@@ -4,6 +4,18 @@
 set -eux
 set -o pipefail
 
+# Check if all symlinks under /dev/disk/ are valid
+helper_check_device_symlinks() {
+    local link
+
+    while read -r link; do
+        if [[ ! -e "$link" ]]; then
+            echo >&2 "ERROR: symlink '$link' points to '$(readlink -f "$link")' which doesn't exist"
+            return 1
+        fi
+    done < <(find /dev/disk -type l)
```

Comment:
Is it possible to also test the symlink target is correct?

Reply:
That crossed my mind as well, but I'm not sure how to determine the correct target. I though about using `udevadm info` on the symlink and then comparing the target to the `$DEVNAME` property. Or is there a better way?

Reply:
I guess that should work. `udevadm info` contains both symlink and device name.